apps/sms/views/shared/js/utils.js

```diff
@@ -473,30 +473,7 @@
       });
       return parsed;
     },
-    /**
-      Using a contact resolver, a function that can looks for contacts,
-      get the format for the dissambiguation.
-
-      Used mainly in activities since they need to pick a contact from just
-      the number.
-
-      In order to workaround facebook contact issue(bug 895817), it should be
-      able to handle the case about phone number without matched contact.
-
-      Phone number comes directly from the activity in the case we call 'pick'
-      from SMS App.
-
-      @param resolver A contact resolver that returns a Promise<{ contacts }>.
-      @param phoneNumber The phone number to retrieve.
-      @returns Promise<Structured Contact>
-    */
-    getContactDisplayInfo: function(resolver, phoneNumber) {
-      return resolver(phoneNumber).then(
-        (contacts) => Utils.basicContact(phoneNumber, contacts)
-      );
-    },
-
-    basicContact: function(number, records) {
+    basicContact: function(number, records, callback) {
```

Comment:
Be careful, you added a parameter here. I think you did it by mistake when rebasing (we removed this useless parameter in a recent patch).